src/vm/docs/mod.rs

```diff
@@ -241,6 +241,25 @@ Note: This function is only available starting with Stacks 2.1.",
 "#,
 };
 
+const IS_STANDARD_API: SimpleFunctionAPI = SimpleFunctionAPI {
+    name: None,
+    signature: "(is-standard principal)",
+    description:
+        "Tests whether `principal` matches the current network type. That is, the network is either
+of type `mainnet`, or `testnet`. And, `principal` is either suited for the
+mainnet or a testnet.  That is, `SPxxxx` and `SMxxxx` addresses are only usable
+on mainnet, whereas `STxxxx` and `SNxxxx` are only usable on testnet and
+regtest. This method will return `true` iff the principal matches the network-type.
```

Comment:
Might want to clarify that no other version bytes are currently spendable.  Also, I don't know if the typical reader knows that `iff` means `if and only if`.